src/extend-utils.ts

```diff
@@ -43,7 +43,7 @@ export async function getContractFactoryUtil(hre: HardhatRuntimeEnvironment, con
         metadataPath,
         abiPath,
         networkID: hre.config.starknet.network,
-        chainID: hre.config.starknet.networkConfig.chainID || TESTNET_CHAIN_ID,
+        chainID: hre.config.starknet.networkConfig.chainId?.toString() || TESTNET_CHAIN_ID,
```

Comment:
Is there a difference between `chainID` and `chainId`? If so, which one should be used here? Writing this note more so to remember to check this, not really sure about the answer myself.

Reply:
[81b7e72](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/77/commits/81b7e72d924c071f07b04ca1a0e2c8ae4919842d)